extensions/ui/org.eclipse.smarthome.ui.paper/README.md

```diff
@@ -18,6 +18,18 @@ The following features are implemented:
 * Configuration of services
 * Event support for item state updates, Thing status updates and new inbox entries
 
+## Development
```

Comment:
Thanks for pointing out. Actually I totally missed that documentation, which was the reason for my PR in the first place. How about linking those documents?